src/main/scala/com/ibm/couchdb/api/builders/GetManyDocumentsQueryBuilder.scala

```diff
@@ -43,87 +57,137 @@ case class GetManyDocumentsQueryBuilder(
           """.stripMargin)
   }
 
-  def conflicts(conflicts: Boolean = true): GetManyDocumentsQueryBuilder = {
-    set("conflicts", conflicts)
+  def conflicts(
+      conflicts: Boolean = true): GetManyDocumentsQueryBuilder[ID, AM, BT] = {
+    set[ID, AM, BT]("conflicts", conflicts)
+  }
+
+  def descending(
+      descending: Boolean = true): GetManyDocumentsQueryBuilder[ID, AM, BT] = {
+    set[ID, AM, BT]("descending", descending)
+  }
+
+  def endKey[K: W](endKey: K): GetManyDocumentsQueryBuilder[ID, AM, BT] = {
+    set[ID, AM, BT]("endkey", write(endKey))
+  }
+
+  def endKeyDocId(
+      endKeyDocId: String): GetManyDocumentsQueryBuilder[ID, AM, BT] = {
+    set[ID, AM, BT]("endkey_docid", endKeyDocId)
+  }
+
+  def includeDocs[D: R]: GetManyDocumentsQueryBuilder[IncludeDocs[D], AM, BT] = {
```

Comment:
- not sure how this would work since the value of the parameter for include docs would influence the return type `IncludeDocs[D]` or `ExcludeDocs` 
- no you would have to use parentheses.
- the type parameter `D` becomes unnecessary for includeDocs(false) as well but this isn't a show stopper